src/components/Tabs/types.ts

```diff
@@ -31,6 +33,11 @@ export interface TabsProps extends BaseComponentProps {
   addButtonElement?: React.ReactElement
   /** Custom Color (Valid color or theme color provided by knit-ui)*/
   customColor?: string
+  /**
+   * As lockToContainerEdges is set `true`, current lockOffset is 0 for react-sortable-hoc, here use can pass
+   * a props & define how much flexible you want your lock offset.
+   */
```

Comment:
```suggestion
  /**
   * As lockToContainerEdges is set `true`, current lockOffset is 0 for react-sortable-hoc, here user can pass
   * a prop & define how much flexibility user wants for lock offset.
   */
```

Reply:
Isn't the default value 50%?

Reply:
yes my bad, but 50% is not working out for us. so set it to 100, which is 100px.

Reply:
making changes in comment